Makefile

```diff
@@ -15,7 +15,7 @@ help:
 # Variables
 #---------------------------------------------------------------------------
 BOX=./.tools/box
-BOX_URL="https://github.com/humbug/box/releases/download/3.8.5/box.phar"
+BOX_URL="https://github.com/humbug/box/releases/download/3.9.0/box.phar"
```

Comment:
it is not available yet: https://github.com/box-project/box/issues/490

Reply:
Yeah, let's wait.